state_synchronizer/src/coordinator.rs

```diff
@@ -108,54 +112,56 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
 
         loop {
             ::futures::select! {
-                msg = self.client_events.select_next_some() => {
-                    match msg {
-                        CoordinatorMessage::Requested(target, subscription) => {
-                            self.request_sync(target, subscription).await;
-                        }
-                        CoordinatorMessage::Commit(version) => {
-                             self.commit(version).await;
-                        }
-                        CoordinatorMessage::GetState(callback) => {
-                            self.get_state(callback);
-                        }
-                    };
-                },
-                (idx, network_event) = network_events.select_next_some() => {
-                    match network_event {
-                        Ok(event) => {
-                            match event {
-                                Event::NewPeer(peer_id) => {
-                                    debug!("[state sync] new peer {}", peer_id);
-                                    self.peer_manager.enable_peer(peer_id, network_senders[idx].clone());
-                                    self.check_progress().await;
-                                }
-                                Event::LostPeer(peer_id) => {
-                                    debug!("[state sync] lost peer {}", peer_id);
-                                    self.peer_manager.disable_peer(&peer_id);
-                                }
-                                Event::Message((peer_id, mut message)) => {
-                                    if message.has_chunk_request() {
-                                        if let Err(err) = self.process_chunk_request(peer_id, message.take_chunk_request()).await {
-                                            error!("[state sync] failed to serve chunk request: {:?}", err);
-                                        }
+                            msg = self.client_events.select_next_some() => {
```

Comment:
a bit confused. What changed here? Is it just indentation and what's a reason for that?

Reply:
not sure what happened there, reverted